Fix to make the buggy moxa work with kermit.

Not intended for upstream, moxa should update their firmware.

Signed-off-by: Wolfram Sang <w.sang@pengutronix.de>
---
 ckctel.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

Index: kermit/ckctel.c
===================================================================
--- kermit.orig/ckctel.c
+++ kermit/ckctel.c
@@ -6628,14 +6628,14 @@ tn_sndcomport()
     /* if (!IS_TELNET()) return(0); */
 
     debug(F100,"tnc_sndcomport","",0);
-    signature = tnc_get_signature();
+    //signature = tnc_get_signature();
     baud = tnc_get_baud();
     datasize = tnc_get_datasize();
     parity = tnc_get_parity();
     stopsize = tnc_get_stopsize();
     oflow = tnc_get_oflow();
     iflow = tnc_get_iflow();
-    tnc_set_ls_mask(255);
+    //tnc_set_ls_mask(255);
     tnc_set_ms_mask(255);
     return(0);
 }
